scripts/build_pkgdown.R

```diff
@@ -35,11 +102,17 @@ for (pkg in packages) {
       warning(paste("No docs folder created for:", pkg))
       next 
     }
-    dest <- file.path("package_documentation/pkgdocs", pkg)
-    if (!dir.exists(dest)) {  
+    pkgname <- desc::desc_get("Package", pkg)
+    dest <- file.path(output_dir, strsplit(pkg, "/")[[1]][1], pkgname)
```

Comment:
I do not think the base / models / modules  arrangement needs to be reflected in the package index -- these folders are mostly for developer convenience and they don't really map to any groupings a PEcAn user will care about. Better just to keep this one flat directory.
```suggestion
    dest <- file.path(output_dir, pkgname)
```